test/failed-directories.test.js

```diff
@@ -124,6 +124,36 @@ describe('cosmiconfig', () => {
       ]);
     });
 
+    it('returns an undefined config, file path, and isEmpty = true for empty rc file', () => {
```

Comment:
Instead of `undefined config, file path, and isEmpty = true`, how about `empty config result`?